Formula/civl.rb

```diff
@@ -0,0 +1,23 @@
+class Civl < Formula
+  desc "The Concurrency Intermediate Verification Language"
+  homepage "http://vsl.cis.udel.edu/civl/"
+  url "http://vsl.cis.udel.edu/lib/sw/civl/1.7/r3157/release/CIVL-1.7_3157.tgz"
+  version "1.7-3157"
+  sha256 "33fa5b4877fbf494b0a4b426c15d6a165fa3db48a9f97cbf391818d965d2dfb9"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8+"
+  depends_on "z3"
+  depends_on "cvc4"
+
+  def install
+    libexec.install "lib/civl-1.7_3157.jar"
+    bin.write_jar_script libexec/"civl-1.7_3157.jar", "civl"
+    share.install "doc", "examples", "licenses", "emacs"
+  end
+
+  test do
+    system "#{bin}/civl", "civl help", "--version", "civl gui"
```

Comment:
What are you trying to do here? After installing `civl`, this is basically equivalent to entering the following on the command line (and I don't think this is what you want):

```
civl "civl help" --version "civl gui"
```

If I try that, I get:

```
CIVL v1.7 of 2016-03-31 -- http://vsl.cis.udel.edu/civl
Command line syntax error at [@0,0:3='civl',<14>,1:0]
Type "civl help" for command line syntax.
```